tests/modules/nf-core/taxpasta/standardise/nextflow.config

```diff
@@ -0,0 +1,10 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: TAXPASTA_STANDARDISE {
+        ext.prefix = { "${meta.id}" }
+        ext.args = { "--profiler ${meta.tool} --output '${task.ext.prefix}.tsv'" }
```

Comment:
Can you remind me if `tsv` is a default? Or must it always be specified via nextflow.config? If the latter might be something to document in the module code

Reply:
There is no default, no. It is parsed from the output file name extension (as above). I'll add a comment.